src/main/common/headers/common.hpp

```diff
@@ -32,7 +32,6 @@
 #endif
 #include <iostream>
 #define SDL2_image  // ? Should we remove this?
-#include <GL/glew.h> //NOLINT
```

Comment:
OpenGL libraries should be fully detached from the rest of the code. Including the header here will cause problems when we want to use Vulkan/DX12.